app/code/Magento/Sales/Controller/AbstractController/Reorder.php

```diff
@@ -57,6 +57,14 @@ public function execute()
         /** @var \Magento\Framework\Controller\Result\Redirect $resultRedirect */
         $resultRedirect = $this->resultRedirectFactory->create();
 
+        $reorderHelper =  $this->_objectManager->get(\Magento\Sales\Helper\Reorder::class);
```

Comment:
Can we move dependencies from Object Manager direct call to the class constructor?

Reply:
Objectmanager was already used in file so i did not add any dependency.

Reply:
Yes, but please move `\Magento\Sales\Helper\Reorder` to the constructor.

Reply:
@swnsma - dependency added. Please review the changes